Source/Cvent.SchemaToPoco.Core/JsonSchemaToPoco.cs

```diff
@@ -152,6 +161,39 @@ private void Generate()
                     Console.WriteLine(entry.Value);
                 }
             }
+
+            if (_configuration.LanguageExportType == LanguageExportType.Python)
+            {
+                CreateInitPyFiles(entryNamespace, moduleList);
+            }
+
+        }
+
+        private void CreateInitPyFiles(string entryNamespace, Dictionary<string, string> moduleList)
+        {
+            var initpyFile = Path.Combine(_configuration.OutputDirectory, entryNamespace.Replace('.', Path.DirectorySeparatorChar), "__init__.py");
+            Console.WriteLine("Wrote " + initpyFile);
+            var directories = entryNamespace.Split(".".ToCharArray());
+
+            string allVar = "__all__ = [";
+            var sep = "";
+            foreach (var module in moduleList)
+            {
+                allVar += sep + module.Key;
+                sep = ",\n" + new String(' ', 11);
+            }
+            IoUtils.GenerateFile(allVar + "]", initpyFile);
+
+            //ignore last directory we just created that file
+            //before this code block.
+            var dirName = "";
+            for (int i = 0; i < directories.Length - 1; i++)
+            {
+                dirName = dirName + "\\" + directories[i];

```

Comment:
`Path.Combine(dirName, directories[i])`
(so this works properly on linux / mono too)